api-compact.yaml

```diff
@@ -60,22 +60,22 @@ auth_header: &auth_header
   type: "string"
 
 limit_qs_parameter: &limit_qs_parameter
-  in: query
-  name: limit
+  in: "query"
+  name: "limit"
```

Comment:
To be like everyone else!  There's a good chance that these values do not need to be quoted, although the examples I've seen have the values quoted, including the prior art in our own code.  I'll try it with unquoted values to see what happens.